modules/system/console/OctoberInstall.php

```diff
@@ -291,7 +311,11 @@ protected function setupMigrateDatabase()
 
         try {
             Db::purge();
-            UpdateManager::instance()->resetNotes()->update();
+
+            UpdateManager::instance()
+                ->setNotesOutput($this->output)
```

Comment:
Those are the last changes in l55upgrade. That is why i wanted to merge this into that branch, for the laravel 5.5 upgrade.
https://github.com/octobercms/october/commit/25bcc3502064d06a854cb4bf829aa3b1a89e28c9#diff-e1f93a69b4e48298ba03a23aef6d3de3